modules/3d/src/usac/ransac_solvers.cpp

```diff
@@ -372,8 +372,8 @@ void UniversalRANSAC::initialize (int state, Ptr<MinimalSolver> &min_solver, Ptr
     } else if (params->isPnP()) {
         degeneracy = makePtr<Degeneracy>();
         if (min_sample_size == 3) {
-            min_solver = P3PSolver::create(points, calib_points, K1);
-            non_min_solver = DLSPnP::create(points, calib_points, K1);
+            min_solver = P3PSolver::create(points, calib_points, K1);   // -->
+            non_min_solver = DLSPnP::create(points, calib_points, K1);  // -->
```

Comment:
Be coherent. Either the DLS method has been silently mapped to EPnP due to an inherent flaw in the conception of the method or because of a bad implementation?
Because here for USAC, it looks like the DLS method is the default method in the general case.

If the DLS implementation in USAC is correct, it should be better to copy this implementation and reactive the SOLVEPNP_DLS codepath in my opinion.